### PR TITLE
Add scopeNote to uniqueText

### PR DESCRIPTION
**Pull request by uscholdm**
Add scopeNote to uniqueText explaining that uniqueness only goes in one direction.

### CODE REVIEW

File: gistCore.ttl [hunk 1]
```diff
@@ -3906,6 +3909,7 @@ gist:uniqueText
 	skos:definition "The unique string value of some content object; i.e., there is no possibility of having more than one value."^^xsd:string ;
 	skos:example "The unique string for a vehicle identification number."^^xsd:string ;
 	skos:prefLabel "unique text"^^xsd:string ;
+	skos:scopeNote "The uniqueness only goes in one direction. A case may arise where two different objects have the same unique text."^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:3912]: This sounds like it is a remote case, but I don't think it is necessarily, given that IDs can represent all sorts of things.

**Review by rjyounes — CHANGES_REQUESTED** [review]

File: gistCore.ttl (hunk 1 above)
Comment (uscholdm) [reply gistCore.ttl:3912]: Can you give a specific and realistic example of two different objects that have the same unique text? I could not think of any. The wording reflected this.

Comment (rjyounes) [reply gistCore.ttl:3912]: Two organizations issue employee IDs. Both employee A at org 1 and employee B at org 2 have ID "12345." I consider this to be a potentially common occurrence.

**Comment by uscholdm** [comment]
Thanks, that's a good example, in principle, but  Im not sure how often o

**Review by rjyounes — APPROVED** [review]

**Pull request merged**